pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1104,6 +1104,15 @@ type KubeletConfig struct {
 	//     - false
 	//     - no
 	KubeletSkipNodeRegistration *bool `yaml:"skipNodeRegistration,omitempty"`
+	//   description: |
+	//     The `enableManifestsDirectory` field configures the kubelet to get static pod manifests from the /etc/kubernetes/manifests directory.
+	//     It's recommended to configure static pods with the "pods" key instead.
+	//   values:
+	//     - true
+	//     - yes
+	//     - false
+	//     - no
+	KubeletEnableManifestsDirectory *bool `yaml:"enableManifestsDirectory,omitempty"`
```

Comment:
for backwards compatibility (when Talos 1.2 is upgraded to Talos 1.3) we need to keep this "on" by default:

* either we call it `DisableManifestsDirectory`
* or we change the `v1alpha1_provider` to make `true` the option when the option is `nil`